src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -6613,43 +6613,43 @@ void FillRemainingPredefinedEntries(EnergyPlusData &state)
         PreDefTableEntry(state,
                          state.dataOutRptPredefined->pdchOaTmFctNoLimit,
                          state.dataAirSystemsData->PrimaryAirSystems(iSys).Name,
-                         state.dataSysRpts->SysPreDefRep(iSys).TimeAtOALimit[MixedAir::limitFactorNone]);
+                         state.dataSysRpts->SysPreDefRep(iSys).TimeAtOALimit[static_cast<int>(MixedAir::OALimitFactor::None)]);
```

Comment:
So TimeAtOALimit is a std::array and you are doing an index lookup using the enum value.  This is fine of course given our current enum paradigm in E+.